src/vip/batch_address_test_tool/processor.clj

```diff
@@ -63,7 +63,10 @@
 
 (defn retrieve-polling-locations*
   [ctx]
-  ctx)
+  (let [addresses (:addresses ctx)
+        polling-locations (doall (map #(civic-info/address->polling-location (:address %)) addresses))
```

Comment:
I recall it not actually throwing the exception when we didn't have the `doall`, but maybe that was before we put in the `mapv`? I recall being surprised that we needed the `doall`, but maybe we should take another look at this, maybe write a test that throws an exception at some point and figure out if we do need the `doall` or not.